builds/dumbo/scripts/controllers/nav.js

```diff
@@ -32,6 +32,12 @@ angular.module('dumboApp')
 				flyout.toggleClass('hidden');
 			});
 
+            $('.autotoggle').each(function(i, e) {
```

Comment:
I needed to add an event handler to some of the header elements to close the navbar since that logic wasn't in place yet. Some of the elements closed the navbar, but the ones that I added .autotoggle to did not close the navbar when clicked.